### PR TITLE
new speech assets + changed the database name in connection

### PR DESCRIPTION
**Pull request by GabeLoins**
@vqtran 
 check it out

### CODE REVIEW

File: bin/connect-to-db [hunk 1]
```diff
@@ -1,3 +1,3 @@
 #!/bin/bash 
 
- mysql -h speechql.cutq0x5qwogl.us-east-1.rds.amazonaws.com -P 3306 -u gabe -p
+ mysql -h speechql.cutq0x5qwogl.us-east-1.rds.amazonaws.com -P 3306 -u gabe -p --local-infile=1
```
Comment (GabeLoins) [inline bin/connect-to-db]: the local infile thing lets you insert cvs on the client machine to the server (by default its not allowed)

Comment (vqtran) [inline bin/connect-to-db]: What does this actually do?

File: src/main/java/echoquery/sql/SingletonConnection.java [hunk 2]
```diff
@@ -15,7 +15,7 @@
 
   private static final String URI = 
       "jdbc:mysql://speechql.cutq0x5qwogl.us-east-1.rds.amazonaws.com:3306/";
-  private static final String DB = "bestbuy";
+  private static final String DB = "airlines";
```
Comment (vqtran) [inline src/main/java/echoquery/sql/SingletonConnection.java]: hmm this breaks our tests which is querying over the much easier bestbuy dataset

File: bin/connect-to-db (hunk 1 above)
Comment (GabeLoins) [inline bin/connect-to-db]: http://stackoverflow.com/questions/14127529/mysql-import-data-from-csv-using-load-data-infile

File: src/main/java/echoquery/sql/SingletonConnection.java (hunk 2 above)
Comment (GabeLoins) [inline src/main/java/echoquery/sql/SingletonConnection.java]: okay maybe I should just merge the new tables into bestbuy db

doesnt really make sense but probably our best bet

Comment (vqtran) [inline src/main/java/echoquery/sql/SingletonConnection.java]: Oh or perhaps maybe just have another connection instance that a AggregationHandler could take in just when doing tests?

**Comment by vqtran** [comment]
I was trying to do what I suggested on this branch but seems like something is messed up with the tests on this branch anyways. So I'm going to merge this, and include the fix with my where clause branch. Everything looks good here otherwise anyways! No need to merge any tables.

**Pull request merged**